go/Dockerfile

```diff
@@ -1,17 +1,19 @@
-FROM ubuntu:14.04
+FROM alpine:3.3
 MAINTAINER Keybase <admin@keybase.io>
 
-RUN apt-get update && apt-get install -y fuse openssl jq
+RUN apk add --update fuse jq bash && \
+    rm -rf /var/cache/apk/* && \
+    mkdir -p /home/keybase && \
+    mkdir /keybase && \
+    adduser -D keybase && \
+    chown keybase:keybase /keybase
 
-RUN useradd -m keybase
-RUN mkdir /keybase
-RUN chown keybase:keybase /keybase
 USER keybase
 WORKDIR /home/keybase
-ADD keybase/keybase /home/keybase/keybase
 
-ENV KEYBASE_DEBUG=0
-ENV KEYBASE_RUN_MODE=devel
-ENV KEYBASE_SERVER_URI="http://kbweb.local:3000"
-ENV KBFS_DYNAMO_SERVER_ADDR="http://dynamodb.kbfs.local:8000"
```

Comment:
We do, but it's KBFS-scoped, so I put it in the `kbfsfuse` container, which gets built from this one.